### PR TITLE
addding documentation and comments

### PR DESCRIPTION
**Pull request by JosueRodriguezGarcia**
Team, I finished of adding the documentation y comments that missing, please check my code.

### CODE REVIEW

File: src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java [hunk 1]
```diff
@@ -8,55 +8,44 @@
...
-    public void setWidth(String width) {
-        Width = width;
-    }
-
+    //The getExt Method return the extension of the output file
     public String getExt() {
         return ext;
     }
 
+    //the setExt Method modify the extension od the output file
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java]: Uppercase at the beginning comment and adding point at the final.

File: src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java [hunk 2]
```diff
@@ -8,55 +8,44 @@
...
-
-    public String getWidth() {
-        return Width;
-    }
-
-    public void setWidth(String width) {
-        Width = width;
-    }
-
+    //The getExt Method return the extension of the output file
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java]: add a point in the final phrase.

File: src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java [hunk 3]
```diff
@@ -8,55 +8,44 @@
...
     public void setName(String name) {
         this.name = name;
     }
 
+    //The getTime Method return the time of the frame that we search.
     public String getTime() {
         return time;
     }
 
+    //The setTime Method modify the time of the frame that we search
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java]: Add a point in the final of the phrase, please.

File: src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java (hunk 1 above)
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java]: The first letter of the comment should be capitalized.

File: src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java [hunk 4]
```diff
@@ -8,55 +8,44 @@
...
-    private String length;
-    private String Width;
     private String ext;
 
+    //The getName method return the name of the output file
     public String getName() {
         return name;
     }
 
+    //The setName method modify the name of the output file
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java]: Add a point in the final of the phrase, please.

File: src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java [hunk 5]
```diff
@@ -8,55 +8,44 @@
...
  */
-public class CriteriaThumbnailVideo extends CriteriaConvert{
+public class CriteriaThumbnailVideo extends CriteriaConvert {
     private String name;
     private String time;
-    private String length;
-    private String Width;
     private String ext;
 
+    //The getName method return the name of the output file
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java]: Add a point in the final of the phrase, please.

File: src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java [hunk 6]
```diff
@@ -8,55 +8,44 @@
  * with Jalasoft.
  */
 package com.fundation.webservice.model;
+
 /**
- * Implements the criteria for create a Thunbnail
+ * Implements the criteria for create a Thumbnail
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java]: Add a point in the final of the phrase, please.

File: src/main/java/com/fundation/webservice/model/ThumbnailVideo.java [hunk 7]
```diff
@@ -25,6 +25,7 @@ public ThumbnailVideo(CriteriaThumbnailVideo criteria) {
         this.criteria = criteria;
     }
 
+    //The convert method capture a thumbnail of a frame specific
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/ThumbnailVideo.java]: Add a point in the final of the phrase, please.

Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/ThumbnailVideo.java]: Write the documentation in third person, please.

File: src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java (hunk 1 above)
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java]: Write the documentation in third person, please.

File: src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java (hunk 2 above)
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java]: Write the documentation in third person, please.

File: src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java (hunk 3 above)
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java]: Write the documentation in third person, please.

File: src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java [hunk 8]
```diff
@@ -8,55 +8,44 @@
...
     public String getName() {
         return name;
     }
 
+    //The setName method modify the name of the output file
     public void setName(String name) {
         this.name = name;
     }
 
+    //The getTime Method return the time of the frame that we search.
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java]: Write the documentation in third person, please.

File: src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java (hunk 4 above)
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java]: Write the documentation in third person, please.

File: src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java (hunk 5 above)
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java]: Write the documentation in third person, please.

File: src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java [hunk 9]
```diff
@@ -20,32 +20,32 @@
...
     }
 
-    //The setTime Method modify the time of the frame that we search.
+    //Modifies the time of the frame that we search.
     public void setTime(String time) {
         this.time = time;
     }
 
-    //The getExt Method return the extension of the output file.
+    //Return the extension of the output file.
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java]: Return? Please use third person.

Comment (JosueRodriguezGarcia) [reply src/main/java/com/fundation/webservice/model/CriteriaThumbnailVideo.java]: ok

**Pull request merged**